File: app/src/main/java/grupo3/rcmm/wifi_indoor_positioning_client/data/home/remote/FingerprintingService.java
package grupo3.rcmm.wifi_indoor_positioning_client.data.home.remote;


import grupo3.rcmm.wifi_indoor_positioning_client.data.home.model.Fingerprinting;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface FingerprintingService {
    @POST("fingerprint")
    Call<Void> postFingerprint(@Body Fingerprinting fingerprint);
}

File: app/src/main/java/grupo3/rcmm/wifi_indoor_positioning_client/data/home/repository/remote/service/FingerprintingService.kt
package grupo3.rcmm.wifi_indoor_positioning_client.data.home.repository.remote.service

import grupo3.rcmm.wifi_indoor_positioning_client.data.home.model.Fingerprinting
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface FingerprintingService {
    @POST("fingerprint")
    fun postFingerprint(@Body fingerprint: Fingerprinting): Call<Void>
}
